.github/workflows/gitops_canary.yaml

```diff
@@ -0,0 +1,207 @@
+      gitops-name:
+        description: Name of the gitops app
+        type: string
+        required: true
+      gitops-values-script:
+        description: Script to run during gitops (update values.yaml)
+        type: string
+      gitops-canary-values-script:
+        description: Script to run during gitops (update canary values.yaml)
+        type: string
+      gitops-master-values-path:
+        description: branch name to take over, or release for production
+        type: string
+      gitops-ingress-path:
+        description: path to ingress
+        type: string
+    secrets:
+      AWS_ASSUME_ROLE:
+        required: true
+      DOCKER_HUB_USERNAME:
+        required: true
+      DOCKER_HUB_PASSWORD:
+        required: true
+      GITOPS_REPO_PRIVATE_KEY:
+        required: true
+      GITHUB_REPOS_PAT:
+        required: false
+      SLACK_WEBHOOK_URL:
+        required: true
+
+permissions:
+  contents: read
+  packages: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+
+    - name: Set variables
+      id: variables
+      run: |
+        echo "repo=${{ inputs.docker-name }}" >> $GITHUB_OUTPUT
+        echo "tag=release-${{ github.ref_name }}" >> $GITHUB_OUTPUT
+        echo "release=${{ github.ref_name }}" >> $GITHUB_OUTPUT
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Login to Docker
+      id: docker-login
+      uses: ovatu/actions/.github/actions/docker-login@main
+      with:
+        docker-hub-username: ${{ secrets.DOCKER_HUB_USERNAME }}
+        docker-hub-password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+        aws-assume-role: ${{ secrets.AWS_ASSUME_ROLE }}
+
+    - name: Build and push
+      uses: docker/build-push-action@v3
+      with:
+        driver: docker
+        file: deploy/kubernetes/docker/Dockerfile
+        platforms: ${{ inputs.docker-platforms }}
+        push: true
+        tags: |
+          ${{ steps.variables.outputs.repo }}:${{ steps.variables.outputs.tag }}
+          ${{ steps.docker-login.outputs.ecr-registry }}/${{ steps.variables.outputs.repo }}:${{ steps.variables.outputs.tag }}
+        build-args: |
+          VERSION=${{ steps.variables.outputs.tag }}
+        secrets: |
+          "github_token=${{ secrets.GITHUB_REPOS_PAT }}"
+
+    - name: Gitops
+      uses: ovatu/actions/.github/actions/gitops-release@main
+      with:
+        name: ${{ inputs.gitops-name }}-${{ steps.variables.outputs.slug }}
+        repository: ${{ inputs.gitops-repo }}
+        repository-ssh-key: ${{ secrets.GITOPS_REPO_PRIVATE_KEY }}
+        input-directory: ${{ inputs.helm-chart }}
+        output-directory: apps/${{ inputs.gitops-name }}/canaries/${{ steps.variables.outputs.slug }}
+        gitops-name: ${{ inputs.gitops-name }}
+        ingress-path: ${{ inputs.gitops-ingress-path }}
```

Comment:
Is this used?

Reply:
We don't need to pass this variable to the release action, removed